vms/platformvm/state/state.go

```diff
@@ -704,6 +708,14 @@ func (s *state) donePrune() error {
 	return s.singletonDB.Put(prunedKey, nil)
 }
 
+func (s *state) VerifyHeightIndex() error {
+	shouldPrune, err := s.shouldPrune()
+	if err != nil || shouldPrune {
+		return block.ErrIndexIncomplete
+	}
+	return nil
```

Comment:
nit: this implementation will change post 1.11.x right? It'll be return nil

Reply:
Correct. We'll modify `vm.go` to return `nil` and remove this function.